api/src/opentrons/protocol_api/labware.py

```diff
@@ -609,20 +609,16 @@ def well(self, idx: Union[int, str]) -> Well:
     @requires_version(2, 0)
     def wells(self, *args: Union[str, int]) -> List[Well]:
         """
-        Accessor function used to generate a list of wells in top -> down,
-        left -> right order. This is representative of moving down `rows` and
-        across `columns` (e.g. 'A1', 'B1', 'C1'...'A2', 'B2', 'C2')
+        Accessor function that generates a list of wells in a top down,
+        left to right order. This is representative of moving down rows and
+        across columns (e.g. ``'A1'``, ``'B1'``, ``'C1'...'A2'``, ``'B2'``, ``'C2'``)
 
         With indexing one can treat it as a typical python
-        list. To access well A1, for example, write: labware.wells()[0]
+        list. To access well A1 for example, write ``labware.wells()[0]``.
```

Comment:
I agree on eliminating the colon.
I like keeping the first comma. But better yet, move `For example, ` to the start of the sentence.
```suggestion
        list. For example, access well A1 with ``labware.wells()[0]``.
```

Reply:
Will fix in separate push.